exercise/exercise_models.py

```diff
@@ -470,6 +470,10 @@ def one_has_deadline_deviation(self, students):
                     deviation = d
         return deviation
 
+    def student_has_deadline_deviation(self, student):
```

Comment:
Is this method really needed? The method `one_has_deadline_deviation` can be called with a list of one student.